KeywordsDetector.java

```diff
@@ -20,7 +20,43 @@ public static void main(String[] args) {
 
     // Iterates through all the sentences.
     // If a sentence contains one or more of the kewords, prints it.
+    
+    public static boolean customContains(String sentences, String keywords) {
+        sentences = sentences.toLowerCase();  
+        keywords = keywords.toLowerCase();
```

Comment:
OK, If it were as a stand alone file 
It is a great helper function.
However this is the same function as `contains` in `MyString` 
`ctrl/command + 'c' , ctrl/command + 'v'`

You could have used `MyString.contains()`